hera_mc/cm_handling.py

```diff
@@ -225,6 +225,45 @@ def show_parts(self, part_dossier, show_args):
                     table_data.append(tdata)
         print('\n' + tabulate(table_data, headers=headers, tablefmt='orgtbl') + '\n')
 
+    def get_specific_connection(self, c, at_date=None):
+        """
+        Finds and returns a list of connections matching the not None components of the query.
```

Comment:
I'm very unclear on what "the not None components of the query" means. Please make this more clear.